Header.c

```diff
@@ -46,6 +46,16 @@ void Header_populateFromSettings(Header* this) {
    Header_forEachColumn(this, col) {
       const MeterColumnSettings* colSettings = &this->settings->columns[col];
       for (int i = 0; i < colSettings->len; i++) {
+         char* end, dynamic[32];
+         unsigned int param = 0;
+         if(String_startsWith(colSettings->names[i], "Dynamic")) {
```

Comment:
Should probably include the opening parens a the name is `Dynamic` for this meter and there's always a parameter in parens expected. This is actually a generic issue with these meters that take additional config. A better fix is probably to allow the meter creation function to return success or failure and skip the meter on failure …

Reply:
> A better fix is probably to allow the meter creation function to return success or failure and skip the meter on failure …

I think it's not relevant at this moment ( ` Header_populateFromSettings` function). at this moment meters are populated from htoprc, not from the meter creation function...

Another idea (I don't think it's ideal) to update htoprc by the meter creation function every ./pcp-htop startup, not only every modification

Reply:
> > A better fix is probably to allow the meter creation function to return success or failure and skip the meter on failure …
> 
> I think this is not relevant at this moment (`Header_populateFromSettings` function). at this moment meters are populated from htoprc, not from the meter creation function...

This is mostly some planning ahead right now: It might only affect the PCP dynamic meters right now, but since some other meters, like the network interface I/O, disk usage, and probably other meters in the future too, will make use of this parameter format, it's likely easier to give the meter a chance to check its parameters and signal any failures. Otherwise we'd be introducing special cases  all over the place without keeping those in the context they apply to.

So my suggested solution would be extending the meter interface to signal back invalid parameters/arguments and have routines creating these meters ignore meters that signal any failures. If any failure was signaled upon loading `htoprc` we flag the config to be saved, thus persisting a clean(ed) copy without the offending meters.

> Another idea (I don't think it's ideal) to update htoprc by the meter creation function every ./pcp-htop startup, not only every modification

See above.